src/screens/EmptyScreen.tsx

```diff
@@ -1,36 +1,30 @@
-import React from "react";
+import React, { useState } from "react";
 import { Dimensions, SafeAreaView, StyleSheet, View, Text } from "react-native";
 import { StackNavigationProp } from "@react-navigation/stack";
-import { getItem, removeItem } from "../services/localStorage";
-import { ButtonPrimary, GenderSelect, InputText } from "../components";
+import {
+  createExecution,
+  timeToString,
+  initializeExecution,
+  cancelExecution,
+  pauseExecution,
+  finishExecution,
+  updateAll,
+} from "../services/timerFunction";
+import { ButtonPrimary, InputText } from "../components";
 
 export interface ScreenProps {
   navigation: StackNavigationProp<any, any>;
 }
 
-const EmptyScreen: React.FC<ScreenProps> = ({ navigation }) => {
-  const eraseRole = async () => {
-    await removeItem("@role");
+const EmptyScreen: React.FC<ScreenProps> = () => {
+  const cardInfo = {
```

Comment:
Só pra gente controlar melhor depois os próximos passos

```suggestion
// TODO: na tela apropriada, pegar dados do paciente atual do AsyncStorage
  const cardInfo = {
```